drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -83,8 +83,14 @@
     transform: translate(-50%, -50%); /* These blocks are to be centered on their absolute x,y position */
 }
 
+/* Focused option */
+.xblock--drag-and-drop .drag-container .item-bank .option:focus,
+.xblock--drag-and-drop .drag-container .item-bank .option:hover {
+    transform: scale(1.1);
```

Comment:
@bradenmacdonald I like the idea of modifying `outline` on `:focus`/`:hover`. The thing I'm worried about is this: What if a course author chooses a background color for items that is similar to the outline color? In that case it would be hard to tell which item currently has focus. Maybe it would make sense to combine the `outline` and `transform` effects on `:focus` (and only use `outline` on `:hover` to make the effect more subtle in that case)? This would also alleviate the problem of items melding together somewhat.

I pushed a version that only includes the changes you are suggesting, but let me know what you think re: combining the effects.

The code for the approach I am suggesting would look something like this:

```css
/* Focused option */
.xblock--drag-and-drop .drag-container .item-bank .option:focus,
.xblock--drag-and-drop .drag-container .item-bank .option:hover {
    outline: 2px solid #fff;
    outline-offset: -4px;
}

.xblock--drag-and-drop .drag-container .item-bank .option:focus {
    transform: scale(1.1);
    box-shadow: 0 16px 32px 0 rgba(0, 0, 0, 0.3);
}
```